package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "plaid",
-  "version": "5.2.0",
+  "version": "5.1.0",
```

Comment:
should this be reverted too?

Reply:
ohh interesting. this was pre-release. getting ready for 6.0